riscv-asm.md

```diff
@@ -183,6 +183,26 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 
+### <a name=.option></a> `.option`
+
+#### `rvc`/`norvc`
+
+Enable/disable C-extension for the following code region.
+
+#### `pic`/`nopic`
+
+Set the code model to PIC (position independent code) or non-PIC, this will
+affect how `la` expands.
+
+#### `relax`/`norelax`
+
+Enable/disable linker relaxation for the following code region if the linker
+supports it.
```

Comment:
"if the linker supports it" -> I think this enables it unconditionally, no? At least Clang will just enable it, and if the linker doesn't support it (LLD...) then it craps out.

I don't know if it's worth mentioning that this also includes emission of`R_RISCV_ALIGN` relocations?